### DISCUSSION TITLE
How to correlate strelka alert to ip / machine

### DISCUSSION THREAD

**Discussion by alkampfergit** [Security-Onion-Solutions/securityonion#2281] [Unsupported Versions] [answered]
Hi,

I'm new to SO, so please forgive me if the question is really simple. I've did some testing transferring from an azure test machine a payload with metasploit encoded with shikata-ga-nai.

It was immediately caught by client antivirus, but I wanted to test if SO correctly find the alert. Now I got a nice strelka alert, really cool.

![image](https://user-images.githubusercontent.com/358545/101663705-53557600-3a4b-11eb-86d5-804867f38a17.png)

From log I can found in /nsm/strelka/processed the file, so I can feed to virus total and everything was good.

What I was not able to do is going directly from this alert to zeek log to verify the ip that downloaded the file. The only way I got is searching with sha1 of the file, it works perfectly, but I wonder if there is a more direct command to go from strelka to all logs related to that file download.

Thanks.

**Comment by lock-wire** [comment, marked as answer]
alkampfergit,
  Zeek extracts the files and assigns a File UID. The field in a Zeek http log for an extracted file will be log.id.resp_fuids.  Strelka logs carry that fuid in the field log.id.fuid.  
  A possible workflow is to start in the Alerts interface.  Left-click the alert and select Hunt.  This pivots you the Hunt Interface.  Expand a Strelka event and left click the value assigned to log.id.fuid.  Select Hunt and pivot to all logs that carry that fuid.  You will get Zeek HTTP logs that carry that fuid.  Now you can use Network Community ID or Zeek log UID to do additional correlation.  

Hope that helps.

**Reply by alkampfergit** [reply]
Thanks a lot, that exactly the info I needed.
